db/seeds.rb

```diff
@@ -10,19 +10,23 @@
 
 apparel = Category.first.items.create([{name: "Hoody",
                       description: "Keeps you slightly warm",
-                      price: 80},
+                      price: 80,
+                      image: File.open('app/assets/images/hoody.jpg')},
 
                      {name: "Sweet Jacket",
                       description: "Keeps you warm",
-                      price: 240}])
+                      price: 240,
+                      image: File.open('app/assets/images/jacket.jpg')}])
 
 
 snowboards = Category.last.items.create([
                     {name: "gwar possum",
                      description: "a snowboard for gwar concerts",
-                     price: 15},
+                     price: 15,
+                     image: File.open('app/assets/images/gwar_possum.jpg')},
 
 
                      {name: "gnar possum",
                       description: "a snowboard for shredding gnar pow",
-                      price: 1000}])
+                      price: 1000,
+                      image: File.open('app/assets/images/gnar_possum.jpg')}])
```

Comment:
Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.